.github/workflows/ci-nymvpn-x-rust.yml

```diff
@@ -7,7 +7,6 @@ on:
       - ".github/workflows/ci-nymvpn-x-rust.yml"
       - "nym-vpn-x/src-tauri/**"
       - "nym-vpn-core/crates/nym-vpn-proto/**"
-      - "nym-vpn-core/crates/nym-gateway-directory/**"
       - "nym-vpn-core/crates/nym-vpnd/**"
```

Comment:
I wonder if we can skip this path trigger too, changes in the daemon should not affect the UI client at this point (exception being proto)

Reply:
yes and proto changes are included as trigger anyway, so useless imo
let's drop it